2-Python Introduction/solution.md

```diff
@@ -0,0 +1,154 @@
+
+10- What does the operator in do?
+
+    in returns True if the specified value is found in the sequence. Otherwise it     returns False.
+
+
+11- What is operator precedence?
+
+    Python operators have a set precedence order, which determines which operators       are evaluated first in a potentially ambiguous expression.
+
+
+12- How is different an expression from a statement? 1.What is an augmented operator? write a code block with an example.
+
+    declarations are anything that can make up a line (or lines) of Python code.     Note that expressions are also statements.
+
+    expressions contain only identifiers, literals, and operators, where operators     include arithmetic and boolean operators, the call operator function of () the     subscription operator [], and the like, and can be reduced to some kind of     "value", which can be any object
+
+
+13- What is string concatenation? write a code block with an example.
+
+    allow you to assign a value to a variable.  example:  r=5; r = r+10
+                                                          print(r)                                                                       
+                                                          answer: 15
+
+14- What is escape sequence and why is it used?
+
+      Concatenation can be defined as the integration of two strings into one       object.     example:  x = "El monto de mi cuenta de banco es de "
+                                                                                                                 y = 100 000
+                                                                                                                 print(x + str (y))     
+                             
+                                                                                                                answer: "El monto de mi cuenta de banco es de 100 000"     
+
+15- What's a formatted strin`? write a code block with an example.
+
+Escape sequences allow you to include special characters in strings. To do this, simply add a backslash (\) before the character you want to escape.
+
+16- Given the string "sinorita", how would you extract all vowels from it using only string indexes?
+
+f-strings are constructed like a normal Python string, but preceded by an f or F. This makes it possible to interpolate code by enclosing it in curly braces ( {} ) within the string. Resulting in a text string.    EXAMPLE:  
+                                                            #Formated Strings
+                                                             name = "Placido"
+                                                             lastname = "Bonilla"
+                                                             age = 21
+                                                             print(f"Hello {name} {lastname}. You are {age} years old and do you like spearfishing\n")
+
+                                                             
```

Comment:
check your answer and questions match and I will continue reviewing. It's difficult to review if questions and answers don't match.